Lombiq.NodeJs.Extensions/CustomExecTasks/MutexAccess.cs

```diff
@@ -0,0 +1,8 @@
+namespace Lombiq.NodeJs.Extensions.CustomExecTasks;
+
+public enum MutexAccess
+{
+    Undefined = 0,
+    Exclusive = 1,
+    Shared = 2,
+}
```

Comment:
```suggestion
public enum MutexAccess
{
    Undefined,
    Exclusive,
    Shared,
}
```

Reply:
I've used both versions in the past. In the current situation the values are not needed, for sure.

They should be used when storing enums as ints e.g. in a DB - then not having immutable values assigned can be a great way to shoot yourself in the foot when you add a new enum member somewhere in the middle (or the beginning 😁).

Reply:
> They should be used when storing enums as ints e.g. in a DB - then not having immutable values assigned can be a great way to shoot yourself in the foot when you add a new enum member somewhere in the middle (or the beginning 😁).

Agree 😄